src/main/java/ColorSupplier.java

```diff
@@ -0,0 +1,7 @@
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        return String.valueOf(Colors.values()[new Random().nextInt(Colors.values().length)]);
```

Comment:
it is optional, but I'd suggest using method toString() instead of String.valueOf()

Reply:
I've tried, and the code hasn't compiled. What is the difference between these two methods?
Bohdan used String.valueOf() method in his example